pkg/meta/meta.go

```diff
@@ -50,8 +50,6 @@ const (
 	AnnotationKeyPropagateFromName      = "from.propagate.crossplane.io/name"
 )
 
-// ReferenceTo returns an object reference to the supplied object, presumed to
-// be of the supplied group, version, and kind.
```

Comment:
Seems like it didn't notice this comment being removed for some reason; I wonder whether it's not using https://github.com/crossplane/crossplane-runtime/blob/master/.golangci.yml.